engine/classes/Elgg/DeprecationService.php

```diff
@@ -65,23 +60,10 @@ function sendNotice($msg, $dep_version, $backtrace_level = 1) {
 		$dep_major_version = (int)$dep_version_arr[0];
 		$dep_minor_version = (int)$dep_version_arr[1];
 
-		$visual = false;
```

Comment:
Are there developers that like current behavior? It annoys the living soul
out of me. But it could just be me.
On Apr 22, 2015 1:05 PM, "Steve Clay" <notifications@github.com> wrote:

> In engine/classes/Elgg/DeprecationService.php
> <https://github.com/Elgg/Elgg/pull/8233#discussion_r28862552>:
>
> > @@ -65,23 +60,10 @@ function sendNotice($msg, $dep_version, $backtrace_level = 1) {
> >  		$dep_major_version = (int)$dep_version_arr[0];
> >  		$dep_minor_version = (int)$dep_version_arr[1];
> >
> > -		$visual = false;
>
> For 2.0 I suggest we put all this info (including stack trace) into an
> array and trigger a hook. Consumers of the hook handle logging/displaying
> an error. Core provides a default handler that just logs the error, to
> duplicate the new behavior in this PR.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8233/files#r28862552>.
>